style.css

```diff
@@ -827,6 +835,7 @@ body {
     height: auto;
     align-items: flex-start;
     justify-content: flex-start;
+    gap: 0.75rem;
   }
 
   .popup-content-container {

```

Comment:
- [ ] [OPTIONAL] Your **popup doesn't have the dark blurred background as it is in the Figma design provided.** It is important to note that this helps to keep your popup in focus so that the user only focuses on the information in the popup window. PLease add that dark blurred background to your design too 😎 🏃

**Your design**
![image](https://user-images.githubusercontent.com/90222110/180575142-21b56f68-4de2-4197-a49e-21c8fb520323.png)

**Figma**
![image](https://user-images.githubusercontent.com/90222110/180575221-29688b01-6baf-4af3-9dde-2631de39783b.png)

Reply:
I can't deploy this feature, because it breaks my popup feature. I hope to do that on weekend after PR approval!!! 😄 I'll have more time and less pressure.